src/plugins/dash/videojs-dash.js

```diff
@@ -94,20 +87,92 @@ class Html5DashJS {
     // - mediaKeyMessageError (only fires under 'might not work' circumstances)
     // eslint-disable-next-line complexity
     this.retriggerError_ = (event) => {
-      if (event.type === 'error') {
-
-        /* initialize errros for ref https://cdn.dashjs.org/latest/jsdoc/core_errors_Errors.js.html
-          map to general init error
-        */
-        if (event.error.code >= 10 && event.error.code <= 35) {
-          this.player.error({ code: 4, dashjsErrorCode: event.error.code, message: event.error.message });
-        } else if (event.error.code < 10) { // network errors
-          this.player.error({ code: event.error.code, dashjsErrorCode: event.error.code, message: event.error.message });
-        } else if (event.error.code >= 100 && event.error.code <= 114) { // Protection Errors  https://cdn.dashjs.org/latest/jsdoc/streaming_protection_errors_ProtectionErrors.js.html
-          this.player.error({ code: 5, dashjsErrorCode: event.error.code, message: event.error.message });
+      if (event.error === 'capability' && event.event === 'mediasource') {
+        // No support for MSE
+        this.player.error({
+          code: 4,
+          message: 'The media cannot be played because it requires a feature ' +
+            'that your browser does not support.'
+        });
+
+      } else if (event.error === 'manifestError' && (
```

Comment:
Maybe we should handle a map object from event id to error code and event message. I think it would be much cleaner instead of all those if.

Reply:
Please open an on-call for the devop on the oncall Slack channel to update the node to v12